src/crimson/common/perf_counters_service.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "common/perf_counters.h"
+#include <seastar/core/sharded.hh>
+
+namespace ceph::common {
+class PerfCountersService: public seastar::sharded<PerfCountersService>
```

Comment:
yeah, `PerfCountersService` is a mini service from the implementation's point of view. but i don't think we should emphasis this in its name. from the user's perspective it's a global collection of perf counters.